app/styles/base.css

```diff
@@ -289,20 +289,24 @@ input[type=button]:active, select:active {
 
 #noVNC_control_bar_handle {
   position: absolute;
-  left: -15px;
+  left: 100%;
+  right: auto;
   top: 0;
   transform: translateY(35px);
-  width: calc(100% + 30px);
+  width: 15px;
   height: 50px;
   z-index: -2;
   cursor: pointer;
-  border-radius: 5px;
+  border-radius: 0px 5px 5px 0px;
   background-color: rgb(83, 99, 122);
   background-image: url("../images/handle_bg.svg");
   background-repeat: no-repeat;
   background-position: right;
   box-shadow: 3px 3px 0px rgba(0, 0, 0, 0.5);
 }
+#noVNC_control_bar_handle:hover {
+  box-shadow: 0px 0px 7px 3px rgba(255, 255, 255, 0.6);
+}
```

Comment:
The reason I added it in the first place was to emphasize that it could be interacted with.. But now as I think more about it, I don't mind removing this.

Another reason to remove it is how poorly hovers work on mobile.